projects/control-service/projects/pipelines_control_service/src/main/resources/application-prod.properties

```diff
@@ -58,6 +58,7 @@ datajobs.kadmin_password=${KADMIN_PASSWORD}
 # This is the full image name of the data job builder
 datajobs.builder.image=registry.hub.docker.com/versatiledatakit/job-builder:1.1
 datajobs.deployment.dataJobBaseImage=registry.hub.docker.com/versatiledatakit/data-job-base-python-3.7:latest
+datajobs.deployment.supportedPythonVersions=${DATAJOBS_DEPLOYMENT_SUPPORTED_PYTHON_VERSIONS:{}}
```

Comment:
Move this to the application.properties since it is the base for all property files.

Reply:
Done, thanks